### DISCUSSION TITLE
Temperature very high on esp32

### DISCUSSION THREAD

**Discussion by Meigl89** [jomjol/AI-on-the-edge-device#3166] [General]
### Issue

Hey everyone,

I'm facing the problem that the temperature of the esp32 is always 76-82°C. I bought one of the recommended esp32s from amazon.de -> https://www.amazon.de/dp/B0CXXYGNL4?psc=1&ref=ppx_yo2ov_dt_b_product_details

I also bought some heat sinks (https://www.amazon.de/dp/B09XB5RNDG?psc=1&ref=ppx_yo2ov_dt_b_product_details) and put it on, but temperature doesn't change. It doesnt feel that warm when I put it in the hand though, but I can't feel the core temperature of the esp32 I guess..

![image](https://github.com/user-attachments/assets/53b946c9-da45-4c3b-adec-cc0e1772a362)

Device info says: CPU cores: 2, Chip revision: 101

Somebody else facing that issue?

In some other post from the past it was that temperature was Fahrenheit instead of Celsius, but that seemed to be the problem on a much older revision.. 

Thanks for any help!

**Comment by SybexX** [comment]
Unfortunately, there are many boards on the market that are of poor quality and such high temperatures are normal for such boards.
A heat sink is only useful if you attach it directly to the chip, because the shielding plate does not sit directly on the chip.
In addition, the camera and the flash LED transfer heat to the chip and some cameras get very hot for some reason.
If you reduce the value of "Wait Before Taking Picture", for example to 2 or 3, you could reduce the temperature by a few degrees.

**Comment by Piti61** [comment]
I've been running three of them for many months: 48°C, 55°C and 95°C. No problems until now.
